cmake/faceliftMacros.cmake

```diff
@@ -405,8 +407,13 @@ macro(_facelift_add_target_start IMPLEMENTATION_TARGET_NAME)
     set(HEADERS_TO_BE_MOCCED ${HEADERS} ${HEADERS_NO_INSTALL})
     unset(HEADERS_MOCS)
 
+    # exclude unnecessary files mocking
```

Comment:
this is not mocking!
e.g. "exclude unnecessary files from running moc"

Reply:
agree, I have noticed that this word doesn't describe enough the purpose. Will be changed.